### ISSUE TITLE
video swap script not working

### ISSUE THREAD

**Issue by benjiTK** [pixeltris/TwitchAdSolutions#287]
Started yesterday, with the script enabled I will get pre-load ads even on subscribed channels. Ads play in the main player, no swapping is happening.

**Comment by pixeltris** [comment]
Can you check the console output for `hookWorkerFetch`? If you don't see this console log the script isn't being injected. If you're using the uBlock Origin script I'd recommend switching to the userscript version due to injection issues mentioned in the README.

**Comment by benjiTK** [comment]
Thank you so much for the reply and help. I have been enjoying the benefits of your project for a long time. 

When I checked the console, I did not see anything for “hookWorkerFetch”. So I disabled the uBlock Origin script and have installed the userscript through ViolentMonkey. Did a restart, and I then checked for "hook" in the console and it says “Multiple twitch adblockers installed. Skipping Worker hook (video-swap-new)”. But pre-load ads seem to still be playing. I have it running to see if I get mid-roll ads too.
